### PR TITLE
Add more AdGuard hosts

### PR DESCRIPTION
**Pull request by anuraag488**
Source
https://github.com/r-a-y/mobile-hosts

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -999,6 +999,33 @@
...
+      "url": "https://raw.githubusercontent.com/r-a-y/mobile-hosts/master/AdguardMobileSpyware.txt",
+      "pack": ["liteprivacy"],
+      "level": [0]
+    },
+    {
+      "vname": "AdGuard Tracking Protection",
+      "group": "Privacy",
+      "subg": "",
+      "format": "hosts",
+      "url": "https://raw.githubusercontent.com/r-a-y/mobile-hosts/master/AdguardTracking.txt",
```
Comment (ignoramous) [inline config.json:1025]: are these mirrors? If so, why not use the direct source?

**Review by ignoramous — COMMENTED** [review]
Thanks, these look decent.

File: config.json (hunk 1 above)
Comment (ignoramous) [reply config.json:1025]: gotcha. i'll review these lists within the next 5 days and get back. cc: @GiddyGoatGaming

Comment (GiddyGoatGaming) [reply config.json:1025]: Seems cool (to me). Might have to test out some of these once they get implemented😉

Comment (ignoramous) [reply config.json:1025]: > I noticed there is already a AdGuard SDNS Filter with format abp used. Will the direct link from AdGuard's other filters will work if same abp format is used?

Our `abp` conversion is wonky. I'd be inclined to replace the existing `abp` list if someone else has the conversion figured out...

Comment (GiddyGoatGaming) [reply config.json:1025]: wally3k has a domain format adguard dns filter [here](https://v.firebog.net/hosts/AdguardDNS.txt) 

Havent seen any complaints about how it works

Comment (r-a-y) [reply config.json:1025]: Wally3k's AdguardDNS filter has less hosts than mine, but not by much.

Comment (ignoramous) [reply config.json:1025]: Thanks. @anuraag488 feel free to replace the source `abp` with r-a-y's (translated) lists

**Comment by ignoramous** [comment]
New blocklists must be added to the end of the json... I can merge this commit as-is.

In my tests, these lists have come good, so I'm inclined to include them.

**Comment by ignoramous** [comment]
AdGuard hosts by r-a-y have now replaced AdAway. Thanks.

Your commit (with the right attribution) has been merged (but this pull-request won't reflect that) @anuraag488: https://github.com/serverless-dns/blocklists/commit/f5039e52416bb72f2eed56e9f1fa0fe68364b835

**Comment by r-a-y** [comment]
I would say that AdAway's hosts file is still good. I would keep it with serverless-dns's blocklists.

**Comment by ignoramous** [comment]
You've have a point but the AdAway list hasn't had a maintainer for 6 months now, and no one has stepped up either, and so sooner or later it is bound to go stale. Btw, projects like OISD / [1Hosts](https://badmojr.github.io/1Hosts/) (which have multiple versions) do include AdAway's list.

**Pull request closed without merging**